src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -157,13 +166,17 @@ def GetClientAccumulatedStats(
         response = messages_pb2.LoadBalancerAccumulatedStatsResponse()
         with _global_lock:
             for method in _SUPPORTED_METHODS:
-                caps_method = _METHOD_CAMEL_TO_CAPS_SNAKE[method]
+                method_path = _METHOD_CAMEL_TO_FULL_PATH[method]
                 response.num_rpcs_started_by_method[
-                    caps_method] = _global_rpcs_started[method]
+                    method_path] = _global_rpcs_started[method]
                 response.num_rpcs_succeeded_by_method[
-                    caps_method] = _global_rpcs_succeeded[method]
+                    method_path] = _global_rpcs_succeeded[method]
                 response.num_rpcs_failed_by_method[
-                    caps_method] = _global_rpcs_failed[method]
+                    method_path] = _global_rpcs_failed[method]
+                response.stats_per_method[
+                    method_path].rpcs_started = _global_rpcs_started[method]
```

Comment:
Not related: YAPF made the line break... syntacticly correct but worse to read.

Reply:
Agreed. Never been a fan of this behavior.